packages/base-map/src/mapControlLocale.tsx

```diff
@@ -0,0 +1,43 @@
+import { IntlShape } from "react-intl";
+
+// Creates a language file to pass to MapLibre that has this format:
+// https://github.com/maplibre/maplibre-gl-js/blob/main/src/ui/default_locale.ts
+
+const MAP_CONTROL_TRANSLATION_KEYS = [
+  "AttributionControl.ToggleAttribution",
+  "AttributionControl.MapFeedback",
+  "FullscreenControl.Enter",
+  "FullscreenControl.Exit",
+  "GeolocateControl.FindMyLocation",
+  "GeolocateControl.LocationNotAvailable",
+  "LogoControl.Title",
+  "NavigationControl.ResetBearing",
+  "NavigationControl.ZoomIn",
+  "NavigationControl.ZoomOut",
+  "ScaleControl.Feet",
+  "ScaleControl.Meters",
+  "ScaleControl.Kilometers",
+  "ScaleControl.Miles",
+  "ScaleControl.NauticalMiles",
+  "TerrainControl.Enable",
+  "TerrainControl.Disable",
+  "CooperativeGesturesHandler.WindowsHelpText",
+  "CooperativeGesturesHandler.MacHelpText",
+  "CooperativeGesturesHandler.MobileHelpText"
+];
+
+const mapControlTranslationObject = (
+  intl: IntlShape
+): { [key: string]: string } => {
```

Comment:
```suggestion
): Record<string, string> => {
```

If you decide to use the suggested syntax, you might need to cast types in the function body.